x-pack/plugins/security_solution_serverless/public/upselling/register_upsellings.tsx

```diff
@@ -124,9 +133,14 @@ export const upsellingSections: UpsellingSections = [
     pli: AppFeatureKey.endpointPolicyProtections,
     component: EndpointPolicyProtectionsLazy,
   },
+  {
+    id: 'endpointExceptions',
```

Comment:
I was confused by this one, the ID looks too generic. If it is only needs to be displayed in the "exceptions" section of the Rules details page, it would be better to use a more specific ID:
```suggestion
    id: 'ruleDetailsEndpointExceptions',
```

Reply:
52d0bc9b88c0016b64f2adf6e11846ebd8b0ab66